v4l2rtspserver-tools/setconf.cpp

```diff
@@ -3,64 +3,149 @@
 #include <cstdlib>
 #include "sharedmem.h"
 
+#define SETGETSHAREDMEMORYINT(INT) if (get) printf("%d\n",  INT); else INT = atoi(value);
+#define SETGETSHAREDMEMORYSTRING(STR) if (get) printf("%s\n",  STR); else  strcpy(STR,value);
+#define SETGETSHAREDMEMORYBOOL(INT) if (get) printf("%s\n",  INT?"TRUE":"FALSE"); else INT= atoi(value)==0?false:true;
+
+int stringToInts(char *str, int region[4])
+{
+    int i = 0;
+
+    char *pt = strtok (str,",");
+    while ((pt != NULL) &&
+            i < sizeof(region)) {
+        int a = atoi(pt);
+        region[i] = a;
+        i++;
+        pt = strtok (NULL, ",");
+    }
+    return (i == 4);
+}
+
+
+void usage(char *command)
+{
+    fprintf(stderr, "Usage %s -g -k KEY -v VALUE\n", command);
+    fprintf(stderr, "Where k to set value, g to get the value\n");
+    fprintf(stderr, "\t'f' flip mode set to\n");
+    fprintf(stderr, "\t\t'1' -> flip on\n");
+    fprintf(stderr, "\t\t'0' -> flip off\n");
+
+    fprintf(stderr, "\t'n' night mode set to\n");
+    fprintf(stderr, "\t\t'1' -> night mode on\n");
+    fprintf(stderr, "\t\t'0' -> night mode off\n");
+
+    fprintf(stderr, "\t'b' set bitrate to VALUE\n");
+    fprintf(stderr, "\t'o' OSD text set to VALUE\n");
+    fprintf(stderr, "\t'c' OSD color set VALUE to\n");
+    fprintf(stderr, "\t\t'0' for White\n");
+    fprintf(stderr, "\t\t'1' for Black\n");
+    fprintf(stderr, "\t\t'2' for Red\n");
+    fprintf(stderr, "\t\t'3' for Green\n");
+    fprintf(stderr, "\t\t'4 for Blue\n");
+    fprintf(stderr, "\t\t'5' for Cyan\n");
+    fprintf(stderr, "\t\t'6' for Yellow\n");
+    fprintf(stderr, "\t\t'7' for Purple\n");
+
+    fprintf(stderr, "\t'x' OSD position Y pos is set to VALUE\n");
+    fprintf(stderr, "\t'p' OSD space between char is set to VALUE (can be negative)\n");
+    fprintf(stderr, "\t'w' fixed text width (0 variable, 1 fixed)\n");
+
+    fprintf(stderr, "\t'm' motion sensibility (0 to 4) -1 to deactivate motion\n");
```

Comment:
Love your contribution. However, can the parameter be called sensitivity please?
The basic distinction is that sensitivity is a fairly universal concept for “readily reacts to someone or something” while sensibility basically means “how practical as a remedy or strategy [a person or plan] is.”

Reply:
Good catch ! I didn't realize that ...
Corrected !